### PR TITLE
add: colors and typography page

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,
I have completed the documentation site for my Component Library. I have made the following changes in this branch.
- Added Colors and Typography page.
- Added README.md
- Added media queries for documentation site

Please review this PR and suggest

### CODE REVIEW

**Review by pradipgharge — COMMENTED** [review]
Looks clean and good to merge 👍🏻

**Review by hsnice16 — APPROVED** [review]
Nice work, Apurva. Dazzle UI has a really neat and nice UI.

File: css/colors.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.palette-container > .color:nth-child(7){
+    background: var(--light-gray);
+    color: black;
+}
+
+.palette-container > .color:nth-child(8){
+    background: var(--dark-gray);
+}
```
Comment (ashishkatiyar2404) [inline css/colors.css:47]: clean code 🙌

**Comment by ashishkatiyar2404** [comment]
Awesome Apurva.... your code looks clean, you have named it very well it's easy to read and understandable to me. ur UI looks 🎉

**Review by mikesch803 — APPROVED** [review]

**Comment by mikesch803** [comment]
clean code 👍

**Review by ashishkatiyar2404 — APPROVED** [review]

**Comment by ApurvaSawant11** [comment]
> clean code 👍

Thank you

**Comment by ApurvaSawant11** [comment]
> Awesome Apurva.... your code looks clean, you have named it very well it's easy to read and understandable to me. ur UI looks 🎉

Thank you so much, Ashish! :)

**Comment by ApurvaSawant11** [comment]
> Nice work, Apurva. Dazzle UI has a really neat and nice UI.

Thank you so much, Himanshu! :)

**Comment by ApurvaSawant11** [comment]
> Looks clean and good to merge 👍🏻

Thank you Pradip!

**Pull request merged**